DiceKeys/Views/Scanning/ScanDiceKey.swift

```diff
@@ -63,61 +64,51 @@ struct ScanDiceKey: View {
-        self.processedImageFrameSize = processedImageFrameSize
-        self.facesRead = facesRead
+        DispatchQueue.main.async {
+            cameraFrameCountModel.frameCount += 1
+            facesReadOverlayModel.facesRead = facesRead ?? []
+            if facesReadOverlayModel.imageFrameSize != processedImageFrameSize {
+                facesReadOverlayModel.imageFrameSize = processedImageFrameSize
+            }
+        }
+
         if facesRead?.count == 25 && facesRead?.allSatisfy({ faceRead in faceRead.errors.count == 0 }) == true {
             try? onDiceKeyRead?(DiceKey(facesRead!).rotatedClockwise90Degrees())
         }
     }
 
     var body: some View {
-            VStack(alignment: .center, spacing: 0) {
-                if cameraAuthorized {
-                    Text("Place the DiceKey so that the \(stickers ? "stickers" : "dice") fill the camera view. Then hold steady.").font(.title2)
-                        .padding(.top, 20)
-                    VStack(alignment: .center, spacing: 0) {
-                        // For debugging: remove
-                        Text("Selected camera: \(selectedOrNextBestCameraDisplayableCamera?.localizedName ?? "nil")")
-                        HStack(alignment: /*@START_MENU_TOKEN@*/.center/*@END_MENU_TOKEN@*/, spacing: 0) {
-                            Spacer()
-                                GeometryReader { reader in
-                                    ZStack {
-                                        DiceKeysCameraView(selectedCamera: selectedOrNextBestCameraDisplayableCamera, onFrameProcessed: onFrameProcessed, size: reader.size)
-                                        FacesReadOverlay(
-                                            renderedSize: reader.size,
-                                            imageFrameSize: processedImageFrameSize ?? reader.size,
-                                            facesRead: self.facesRead
-                                        )
-                                    }
-                                }.aspectRatio(1, contentMode: .fit)
-                            Spacer()
-                        }
-                        if activeCameras.count > 1 {
-                            Picker(
-                                selection: $selectedCameraUniqueId,
-                                label: Text("Camera")
-                            ) {
-                                ForEach(activeCameras) { camera in
-                                    Text(camera.localizedName).tag(camera.ID)
+        VStack(alignment: .center, spacing: 0) {
+            if cameraAuthorized {
+                Text("Place the DiceKey so that the \(stickers ? "stickers" : "dice") fill the camera view. Then hold steady.").font(.title2)
+                VStack(alignment: .center, spacing: 0) {
+                    // For debugging: remove
+                    Text("Selected camera: \(selectedOrNextBestCameraDisplayableCamera?.localizedName ?? "nil")")
```

Comment:
// For debugging: remove

The line below may be a a remnant of code to test cameras.  Remove?